flink-runtime/src/main/java/org/apache/flink/runtime/state/KeyedStateBackend.java

```diff
@@ -37,6 +39,12 @@
 	void setCurrentKey(K newKey);
 
 	/**
+	 * @return Stream of existing keys in the namespace.
```

Comment:
Nit: This could be changed to "Returns a stream of all keys for the given state and namespace.". The first parameter should be renamed to `state` or something like it because `field` is not used anywhere else, I think.